src/store/Store.js

```diff
@@ -31,15 +31,19 @@ class Store {
     if (typeof patchStrategy !== 'function') {
       throw new Error('patchStrategy must be one of the included patching strategies or a custom patching function');
     }
+    
+    const isChrome = /Chrome/.test(navigator.userAgent) && /Google Inc/.test(navigator.vendor);
+
+    const allGlobal= isChrome === true ? chrome : browser;
```

Comment:
Not a huge fan of this name. We should use a more descriptive name of what we're referencing here (which is the host browser api)